### PR TITLE
Pre-merge cleanup

### PR DESCRIPTION
**Pull request by hlinnaka**
These commits do misc cleanup of the codebase, to reduce the diff vs. upstream. These are changes that I had done on the merge_8_3_22 branch me and Daniel are working on, to make the merging easier, but that don't depend on the merge per se.

### CODE REVIEW

File: src/include/utils/sharedsnapshot.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/*-------------------------------------------------------------------------
+ *
+ * sharedsnapshot.h
+ *	  GPDB shared snapshot management
+ *
+ * Portions Copyright (c) 1996-2008, PostgreSQL Global Development Group
+ * Portions Copyright (c) 1994, Regents of the University of California
```
Comment (danielgustafsson) [inline src/include/utils/sharedsnapshot.h]: Shouldn't this have a `Portions Copyright (c) 200X-2016, Pivotal Inc.` header as well?

**Comment by danielgustafsson** [comment]
I'm somewhat biased so more eyes are needed but :+1: from me.

**Comment by kuien** [comment]
:+1: for me. I scan by topdown, so next person may scan by bottom-up, because it gets harder to concentrate on as time going, 😆 

@eurekaka does these code affect the gpfaultinjector utilities? It is a cool tool, but dstrace you demo us is much prettier.

**Comment by hlinnaka** [comment]
Pushed, thanks for the review!

**Pull request merged**